### ISSUE TITLE
will PropertySourcedRequestMappingHandlerMapping impact performance?

### ISSUE THREAD

**Issue by fangjian0423** [springfox/springfox#2336]
Swagger2DocumentationConfiguration will create PropertySourcedRequestMappingHandlerMapping and the order of it is "Ordered.HIGHEST_PRECEDENCE + 1000"。

When i using springboot, the sequence of HandlerMapping in DispatcherServlet will be:

1. PropertySourcedRequestMappingHandlerMapping
2. RequestMappingHandlerMapping
3. WebMvcConfigurationSupport$EmptyHandlerMapping
4. .....

will PropertySourcedRequestMappingHandlerMapping impact performance?

**Comment by fangjian0423** [comment]
![2018-04-02 5 56 05](https://user-images.githubusercontent.com/5254035/38458032-40e4606c-3acb-11e8-9b0e-03d66aaba657.png)

This is the handlerMappings in DispatcherServlet.

PropertySourcedRequestMappingHandlerMapping is the first one.

when i request "/person/save", PropertySourcedRequestMappingHandlerMapping will be the first handlerMapping to handle the request, but it can not get HandlerExecutionChain. so next handlerMapping RequestMappingHandlerMapping will handle it.

**Comment by dilipkrish** [comment]
Yes, there will be a chain, but it should be relatively cheap to make that part of the chain.

**Comment by fangjian0423** [comment]
So PropertySourcedRequestMappingHandlerMapping impact performance, but just a little.

Thanks for answer my question.
